docs/index.md

```diff
@@ -3,14 +3,106 @@ title: "edi3 Inter Customs Ledger 1.0 Specification"
 specID: "icl/1"
 status: "![raw](http://rfc.unprotocols.org/spec:2/COSS/raw.svg)"
 editors: "[Chris Gough](https://github.com/monkeypants)"
-contributors: 
+contributors: "[Richard Spellman](https://github.com/arpentnoir)"
 ---
 
 ## Introduction
+International trade clearance procedures require a number of documents to be presented to satisfy the requirements of a variety of agreements. For example, in order for an importer to gain preferential tariff treatment under a Free Trade Agreement, the importer must present a valid Certificate of Origin (issued to the exporter) which states that the goods being imported qualify as originating in the country which is party to the agreement. These processes are managed largely by the transfer of paper documents which are subject to loss, alteration and forgery. 
+> FIXME
+there should be some text here describing the financial consequences of the last statement above
 
 
+> FIXME
+clear and consise description of the problem that an ICL proposes to solve
+e.g. Clearance of shipments can be delayed because of uncertainty around accompanying documentation, causing signifant costs to bla bla bla
+
+An Inter Customs Ledger is proposed as an apporach to the transfer of shipment related documents between customs agencies which is immediate, permanent and irrefutable.
+
+This document uses the case of managing a Certificate of Origin as the guiding use case for the initial specification, however it is intended that the Inter Customs Ledger should be generic and allow for the transfer of any shipment related documentation between authorised agencies. 
+
+#### Background
+The following present some existing approaches to implementing a blockchain solution to managing certificates of origin, as well as some general trade related blockchain initiatives, and non-blockchain related solutions to solving some of the problems of international shipment documentation. 
+
+> FIXME
+put brief summary of existing certificate of origin solutions and a couple of other things here. short paragraph. reference link below
+
+[Survey of Blockchain Based Approaches to Managing International Trade Documentation](precedents.md)
+
+#### Users
+> FIXME
+list of parties and their roles etc
+exporter - applies for certificate
+authorised body - issues certificate
+importer - applies for preferential tariff treatment
+customs authority - grants preferential tariff treatment
+
+
+#### Example Scenarios
+> FIXME
+> Get full list of documents required for a simple case where only additional 
+> documentation beyond generic shipment documentation is a Certificate of Origin
+> maybe a wine shipment? 
+> as well as all documents required for a more complex scenario, maybe orchids (certificate of origin, CITES, phytosanitary...) as well as full list of parties involved
+
```

Comment:
Maybe we should create non-normative documents (not part of the spec per-se) documenting scenarios, and reference them from this document.

ticket needed: The spec itself does need a scoping statement re: protocol and payloads
 * are we talking about artefacts that are linked to a single export event and a single import event? or is that just the simple case...
 * is there a consistent signalling pattern {approved, exported, imported} or is this one of multiple supported patterns?

Reply:
see #12